### DISCUSSION TITLE
Using Chuck's Dynamic FPS 1.5.5 Beta 3, Which Quality Reduction Graphics mods should be used?

### DISCUSSION THREAD

**Discussion by jaycc3000** [HolographicWings/TOTK-Mods-collection#143] [General]
I am loving the performance boost with Dynamic FPS, however, even with 5120x2880 (textures?), the image isn't very clear/sharp. I tried Yuzu AA, but was wondering what the difference between "Disable Quality Reduction" and the "Disable Internal FSR Downscaling" & "Disable Internal FXAA" Additionally, how does "LOD Improvement" and "Island Fix" work with Dynamic FPS?

**Comment by jaycc3000** [comment]
I wanted to add, on my system, running all of them does seem to significantly improve visuals, but at the cost of about 10 fps

**Comment by hoverbike1** [comment]
Graphics - Disable Internal FSR Downscaling: This mod disables the game's internal FSR. Do not mistake with Yuzu's own FSR on the "Window Adapting Filter" in Graphics settings.
FSR is an optimisation method that upscales the game resolution using several algorithms (similar to DLSS but without advanced AI or temporal information). It can increase GPU performance at the cost of some details and potential graphical artifacts.
Disable FSR if you are using better methods like Yuzu's FSR or even better: using real improved resolution.

Graphics - Disable Internal FXAA v2: Disables TotK's internal FXAA.
FXAA smoothes edges to avoid aliasing (jagged edges) and costs very little GPU performance.
If you want a better anti aliasing, we recommend using SMAA on the "Anti-Aliasing Method" in Graphics settings (at a slightly cost on GPU performance). Using a resolution above your monitor can also work as anti-alising, but have high cost on your GPU (see the Resolution section).
❗Overwrites "Bootup.Nin_NX_NVN.pack.zs", don't install two mods overwriting the same file!

Graphics - LOD Improvement: Increases the Level of Detail on certain models.
LOD (Level Of Detail) shows higher quality models when you are close to them, and lower quality models when they are further away.
Has a minor performance improvement (we spent days testing on different hardware to believe it). Our theory is that less model swappings helps the emulator because there's less things changing at the same time.
[Comparison On/Off](https://imgsli.com/MTgyMzE5)

Graphics - Disable Quality Reduction: Prevents graphics quality decreasing when your framerate is under 30fps.
LOD (Level Of Detail) shows higher quality models when you are close to them, and lower quality models when they are further away.
By default, when your framerate is low, the game lowers the LOD to increase GPU performance, causing textures and models to visibly get worse.
When this happens the game also reduce the resolution, causing pixelated images.
BEWARE! Mandatory if using Chucks Resolution Mods (Or another Pchtxt resolution Mod). Not necessary with Sweetmini resolution Mods, especially if you use LOD Improvement Mod.

https://github.com/HolographicWings/TOTK-Mods-collection

Its right there, please read it, writing it took long enough.
I dont recommend you use anything except LOD improvement and Disable Quality reduction with DFPS v155.
